Assets/NuGet/Editor/NugetConfigFile.cs

```diff
@@ -76,6 +77,14 @@ public void Save(string filepath)
                     addElement.Add(new XAttribute("value", "true"));
                     disabledPackageSources.Add(addElement);
                 }
+
+                if (source.HasPassword)
+                {
+                    addElement = new XElement("add");
+                    addElement.Add(new XAttribute("key", source.Name));
```

Comment:
Agreed - we should move this over to the standard.

Reply:
Fixed.